lucene/spatial-extras/ivy.xml

```diff
@@ -29,6 +29,8 @@
       <artifact name="spatial4j" type="test" ext="jar" maven:classifier="tests" />
     </dependency>
 
+    <dependency org="io.sgr" name="s2-geometry-library-java" rev="1.0.0" conf="test"/>
```

Comment:
Instead you're supposed to reference the version via ivy-versions.properties.  Also run "ant jar-checksums" to add the checksum file.